src/Model/CambiaStato.php

```diff
@@ -0,0 +1,12 @@
+<?php
+class CambiaStato {
+
+  private $stato;
+
+  public function switch() {
```

Comment:
immagino che questa funzione avrebbe il compito di cambiare lo stato di un ordine. Potrebbe quindi ricevere in input un Ordine e  uno stato e aggiornare lo stato, ritornando un booleano che indichi se l'operazione è riuscita